### PR TITLE
Silent renew

### PR DESCRIPTION
**Pull request by shaunluttin**

### CODE REVIEW

**Comment by shaunluttin** [comment]
The works on my machine.
``` 
git clean -xfd
cd samples\ImplicitFlow
runDemo.ps1
```
I would welcome a review, at your leisure and convenience, @PinpointTownes.

https://www.youtube.com/watch?v=fL5N8rSy4CU

**Comment by PinpointTownes** [comment]
@shaunluttin thanks for the PR!

When you have a minute, I'd love to chat with you on Gitter about this PR.

**Comment by PinpointTownes** [comment]
@shaunluttin can I merge it? (I'll add the server-side part before pushing it)

File: samples/ImplicitFlow/AuthorizationServer/Properties/launchSettings.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:12345",
+      "applicationUrl": "http://localhost:5000",
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AuthorizationServer/Properties/launchSettings.json]: Curious: why changing the port?

Comment (shaunluttin) [inline samples/ImplicitFlow/AuthorizationServer/Properties/launchSettings.json]: `5000` is so common that it interfered with other apps that might also be running.

**Comment by shaunluttin** [comment]
@PinpointTownes Sure. Go ahead and merge it. Apologies for dropping the ball on this.

**Comment by PinpointTownes** [comment]
Merged, thanks! :+1: 

https://github.com/openiddict/openiddict-samples/commit/1fd271e6aa23ad8afb09d9cb9a422148df67d8a0

**Pull request closed without merging**